lib/viewport.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+
+class FixedVerticalResolutionViewport extends Viewport {
+  late double effectiveHeight;
```

Comment:
This can be final right? It is being set in the constructor so at least it does not have to be `late`.